src/views/products.vue

```diff
@@ -179,4 +178,13 @@ export default defineComponent({
   max-height: 450px;
 }
 
+.features{
```

Comment:
@dt2patel 
Could you please review the styling changes?

Reply:
@disha1202 I think its ok if we present the feature value as it comes from the server. is there a reason you decided to transform it?

Reply:
Previously we were hard coding the attribute names(Size and Color), we used sentence case for that.
And from the server we get it in all caps, that's why we've transformed it.